src/args.py

```diff
@@ -21,14 +29,17 @@
 a = [7, 6, 5, 4]
 
 # What thing do you have to add to make this work?
-print(f2(a))    # Should print 22
+print(f2(sum(a)))    # Should print 22
```

Comment:
Some students are using print(f2(*a)) instead, the * is the splat operator, which spreads out the arguments so you can sum them.